Translate v2/guide partially

rgomez90 opened the pull request:
- v2/guide/custom-directive
- v2/guide/components
- v2/guide/computed
- v2/guide/ssr
- v2/guide/plugins
- v2/guide/routing
- v2/guide/migration-vuex
- v2/guide/deployment
- v2/guide/filters
- v2/guide/joing

miljan-aleksic commented:
Todo parece correcto, sigue añadiendo segun avances. Si hay algo que creas que require revision mas detallada, dimelo.

rgomez90 commented:
@miljan-aleksic en el archivo `team` traduzco los comentarios y el contenido de los tags "visibles", sólo el contenido o no traduzco nada? Parece que ese archivo es sólo HTML y JS por lo que supongo que los comments no estan ahí para "el público".

miljan-aleksic commented:
El script de esa pagina son partes de la ejecucion, no ejemplos. No los traduzcas, tan solo lo visible. Puedes ver lo que han traducido en [Frances](https://fr.vuejs.org/v2/guide/team.html), basicamente los titulos de cada miembro.

rgomez90 commented:
@miljan-aleksic echale un ojo al [commit translate v2/guide/team](https://github.com/vuejs-es/vuejs.org/pull/53/commits/cdcd18d5f296fc431edea04a61c6670ab3a9e430)

miljan-aleksic commented:
Parece correcto :)

miljan-aleksic commented on src/v2/guide/unit-testing.md:
```diff
@@ -31,15 +31,15 @@ No es necesario hacer algo en especial para poder probar sus componentes. Export
-// Éstas son algunas pruebas con Jasmine 2.0, pero puede usar
-// cualquier ejecutor de pruebas y librería de aserción de su preferencia
+// Éstas son algunos tests con Jasmine 2.0, pero puede usar
+// cualquier ejecutor de pruebas y librería de assert de su preferencia
```
Assert, afirmacion, comprobar? Parece que se mezclan los terminos. Votamos y los añadimos al Wiki?

rgomez90 replied:
sería lo más lógico

miljan-aleksic replied:
Te puedes encargar de ellos en el Issue #1?

miljan-aleksic commented:
@rgomez90, ya tenemos traducida la API. Que te falta por terminar para ya revisar este commit?

rgomez90 commented:
Para este finde lo tengo listo que he estado 3 semanas de vacaciones desconectado de este mundo.

miljan-aleksic commented:
@rgomez90, perdona que te insista. Es lo ultimo que nos falta para dar por terminada la traduccion y empezar la revision general. Si necesitas ayuda para terminarlo podemos hacer un merge de lo que tienes y ayudarte con lo que falte.

rgomez90 commented:
Buenas @miljan-aleksic, disculpa la ausencia, pero estoy liadisimo. Esta tarde pongo mi "status", en cuanto llegue del trabajo.

rgomez90 commented:
@miljan-aleksic alguien podria traducir el archivo `v2/guide/comparison` que no lo empecé y tiene para rato.

rgomez90 commented:
@miljan-aleksic sabes si juansaab terminó de traducir `transitions` y `migrations` de `src/v2/guide/`?

miljan-aleksic commented:
@rgomez90, no estoy seguro, tenemos que hacer una revision general de lo que queda. Propongo hagamos un merge de lo que tengas hecho y asignemos tareas de lo que falte.

Deberias de hacer un nuevo Pull Request en el branch master, que es el que tiene todo el history y los ultimos cambios. Simplemente clona el repositorio de nuevo, aplica por encima manualmente tus cambios (tan solo de las carpeta guide), commit y pull request. Con ello todos estariamos a la par en master y podremos seguir adelante.

Te parece bien?

rgomez90 commented:
Perfecto. A ver si soy capaz xD, que en Git todavia tengo el nivel Noob 😆 😆

rgomez90 committed:
Partial translation v2/guide

- v2/guide/class-and-style
- v2/guide/conditional
Translate custom-directive & components

Translate v2/guide/computed

Translate v2/guide/routing

Translte v2/guide/ssr

Translate v2/guide/plugins

translate v2/guide/filters

translate v2/guide/join

Update Data bindings

update data bindings

fixed custom-directive

Translate v2/guide/deployment

Translate v2/guide/migration-vuex

Translate v2/guide/forms

Update v2/guide/join

translate v2/guide/unit-testing

translate v2/guide/reactivity

translate v2/guide/single-file-components

translate v2/guide/state-management

translate v2/guide/syntax

translate v2/guide/team

Update unit-testing to implement #59

Translate v2/guide/index


Fix class and style


Translate /v2/guide/events


partial translation v2/guide/installation

rgomez90 commented:
@miljan-aleksic antes de liarla parda...me puedes explicar un poquillo mejor el proceso? Simplemente clone el repo de nuevo y sobreescribi los archivos de `guide` que habia editado y commit. Pero ahora como hago un pull request sin haber hecho primero un fork, sino haber simplemente clonado el repo?

Si puedes explicarme quizas mejor por aqui (un chat temporal que abri para el caso) [http://www.disposablechat.com/chat/vuejses?password=vuejs](http://www.disposablechat.com/chat/vuejses?password=vuejs)

miljan-aleksic commented:
Cierro este ya que se ha creado y aceptado desde uno nuevo.
